content/_index.md

```diff
@@ -27,31 +28,18 @@ images:
 		<img src="https://img.shields.io/github/license/odin-lang/odin.svg">
 	</a>
 </p>
-</center>
+</section>
 
-# The Odin Programming Language
+# Design Goals
```

Comment:
I don't think the title needs to be changed to _Design Goals_ here.

Reply:
I think the homepage needs to be reworked anyway, don't you?

We don't want to say "simple", we want to show or explain why it's simple.

Don't say performant, show a benchmark.

Don't say joyful, show how, etc

Reply:
I agree. Showing by example is much better than an assertion.